### DISCUSSION TITLE
Add offline installer feature to osu! for players who can't install for internet reasons

### DISCUSSION THREAD

**Discussion by heerxingen** [ppy/osu#26757] [Ideas] [answered]
Is it possible to make an offline installer for osu! ? Players in China usually has problems with installing with the normal installer due to the GFW, although this can be fixed with a VPN but they are expensive to buy and some players don't even know that much about technology. In the Chinese player community many people have complained:'I spend three hours waiting for the installer to install the game but nothing happened, but after I used a vpn it installed like in three minutes' or something similar
I really hope that osu could make a offline installer to benifit players who has bad internet connections or just want to use a offline installer for some reason.

**Comment by cihe13375** [comment]
the lazer installer (in the release page) already works offline I think

**Reply by peppy** [reply, marked as answer]
yep, the install is already "offline".

**Comment by heerxingen** [comment]
yes I know but I mean the stable version
